src/docs/development/androidx-migration.md

```diff
@@ -82,7 +82,7 @@ steps in previous section.
 
 * **Apps and Modules**
```

Comment:
This still references Modules.

Reply:
The project is still a module, but it's used in the context of add to app.